components/payments/Earthfile

```diff
@@ -108,6 +108,15 @@ tidy:
     WORKDIR /src/components/payments
     DO --pass-args stack+GO_TIDY
 
+generate:
+    FROM core+builder-image
+    RUN apk update && apk add openjdk11
+    DO --pass-args core+GO_INSTALL --package=go.uber.org/mock/mockgen@latest
```

Comment:
I have a question here:why changing the gomock package from github.com/golang/mock to go.uber.org/mock/mockgen?

Reply:
My bad, as @gfyrag said, the former is deprecated !